spot-ml/src/main/scala/org/apache/spot/netflow/model/FlowScoreFunction.scala

```diff
@@ -0,0 +1,70 @@
+package org.apache.spot.netflow.model
+
+import org.apache.spark.broadcast.Broadcast
+import org.apache.spot.SuspiciousConnectsScoreFunction
+import org.apache.spot.netflow.{FlowWordCreator, FlowWords}
+
+
+/**
+  * Estimate the probabilities of network events using a [[FlowSuspiciousConnectsModel]]
+  * @param timeCuts
+  * @param ibytCuts
+  * @param ipktCuts
+  * @param topicCount
+  * @param ipToTopicMixBC
+  * @param wordToPerTopicProbBC
+  */
+
+class FlowScoreFunction(timeCuts: Array[Double],
+                        ibytCuts: Array[Double],
+                        ipktCuts: Array[Double],
+                       topicCount: Int,
+                       ipToTopicMixBC: Broadcast[Map[String, Array[Double]]],
+                       wordToPerTopicProbBC: Broadcast[Map[String, Array[Double]]]) extends Serializable {
+
+
+  val suspiciousConnectsScoreFunction =
+    new SuspiciousConnectsScoreFunction(topicCount, ipToTopicMixBC, wordToPerTopicProbBC)
```

Comment:
As pero last days working sessions, broadcasting of datasets larger than 10 MB should never happen again. We need to revisit the implementation for word to topic mix lookup too as well.